app/src/main/java/org/gdg/frisbee/android/chapter/InfoFragment.java

```diff
@@ -124,28 +93,38 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
         final String chapterPlusId = getArguments().getString(Const.EXTRA_PLUS_ID);
         if (Utils.isOnline(getActivity())) {
-            new Builder<>(String.class, Person.class)
-                .setOnBackgroundExecuteListener(new CommonAsyncTask.OnBackgroundExecuteListener<String, Person>() {
+            App.getInstance().getModelCache().getAsync(Const.CACHE_KEY_PERSON + chapterPlusId,
```

Comment:
This line now exists in the else statement as well [here](https://github.com/gdg-x/frisbee/pull/589/files#diff-42ae8acf20f2f4d28c1a2703a94e0c2bR129)

I don't think this `isOnline` check is helpful. Because of that we have a hell of indentation here. I think shouldn't have `getAsync` from model 2 times. 

What we can do is that, we can first try to get it online, if that is unsuccesful, we can try to get from the cache, if that fails, we can use `setIsloading(false)`